Header header
uint32 node_id
string state

string STATE_START='start'
string STATE_NEXT='next'
string STATE_END='end'
